### PR TITLE
WRQ-25915: Added custom site icon for sampler app

### PR DESCRIPTION
**Pull request by daniel-stoian-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
In storybook 8, the favicon.ico file available on the server is not loaded. the storybook default icon is rendered.
This is caused by an API change in storybook8

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Storybook 8 loads a "favicon.svg" after build:

image

In storybook 6, this explicit use was not generated, but it would use any available favicon.ico from the server.

the server where we deploy storybook has a favicon.ico available at the root level. 
My solution was to add the enact icon in the storybook public folder and adapt storybook configuration to use the custom icon.

According to the communitty, in order to work, the custom file must be named exactly favicon.ico https://github.com/storybookjs/storybook/issues/20487 

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRQ-25915

### Comments

Enact-DCO-1.0-Signed-off-by: Daniel Stoian ([daniel.stoian@lgepartner.com](mailto:daniel.stoian@lgepartner.com))

### CODE REVIEW

File: packages/sampler/.storybook/manager-head.html
Comment (stanca-pop-lgp) [inline packages/sampler/.storybook/manager-head.html:1]: Please add empty line at the end of file.

**Review by stanca-pop-lgp — CHANGES_REQUESTED** [review]

File: packages/sampler/.storybook/manager-head.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<link rel="shortcut icon" href="/favicon.ico" />
```
Comment (daniel-stoian-lgp) [inline packages/sampler/.storybook/manager-head.html:1]:
```suggestion
<link rel="shortcut icon" href="/favicon.ico" />

```

**Review by MikyungKim — APPROVED** [review]
LGTM

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**